execute/execute.py

```diff
@@ -0,0 +1,162 @@
+import argparse
+import json
+import jsonschema
+import time
+
+from pathlib import Path
+
+from console_access_api.aitrios_console import AitriosConsole
+
+parser = argparse.ArgumentParser(
+    description="Launches execution on a camera configured in AITRIOS Console",
+    formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+)
+
+parser.add_argument(
+    "--aitrios_secrets",
+    "-s",
+    type=str,
+    dest="aitrios_secrets",
+    help=f"The path to your AITIRIOS console configuration file",
```

Comment:
Typo, unnecessary f-string

```suggestion
    help="The path to your AITRIOS console configuration file",
```